kolibri/plugins/coach/assets/src/views/elapsed-time/index.vue

```diff
@@ -0,0 +1,99 @@
+<template>
+
+  <span>{{ $tr($trUnit, {n: display}) }}</span>
```

Comment:
Woulding skipping `$trUnit` and passing the the time unit in as a `select` option make for less translation strings? Like I had been doing [here](https://github.com/learningequality/kolibri/pull/1201/files#diff-9be3a6fbabe95ded3bf719718a9a3163L40)? 

Does this look better from a translator's perspective? Or do you consider it just easier to makes sense of as a dev?